### PR TITLE
Improve/Fix CardRefresh

### PR DESCRIPTION
**Pull request by hkvstore**
- Fix ``_jQueryInterface()``
- Fix doc
- Use ``$.ajax()`` instead of ``$.get()`` and add ``ajaxSettings`` option for the ``load()`` method

### CODE REVIEW

**Comment by danny007in** [comment]
is this PR an breaking change

**Comment by danny007in** [comment]
can it fix https://github.com/ColorlibHQ/AdminLTE/issues/3851

**Comment by hkvstore** [comment]
> is this PR an breaking change?

No.

**Comment by hkvstore** [comment]
> can it fix #3851

Yes.

File: build/js/CardRefresh.js [hunk 1]
```diff
@@ -65,18 +66,22 @@ class CardRefresh {
...
-        }
+    $.ajax(this._settings.source, $.extend({
+      data: this._settings.params,
+      success: response => {
+        if (this._settings.loadInContent) {
+          if (this._settings.sourceSelector !== '') {
+            response = $(response).find(this._settings.sourceSelector).html()
+          }
 
-        this._parent.find(this._settings.content).html(response)
```
Comment (art300939) [inline build/js/CardRefresh.js:74]: #
Duplicate of #

File: build/js/CardRefresh.js [hunk 2]
```diff
@@ -65,18 +66,22 @@ class CardRefresh {
...
-      }
```
Comment (art300939) [inline build/js/CardRefresh.js:75]: #3514

File: build/js/CardRefresh.js [hunk 3]
```diff
@@ -65,18 +66,22 @@ class CardRefresh {
...
+          this._parent.find(this._settings.content).html(response)
+        }
 
-      this._settings.onLoadDone.call($(this), response)
```
Comment (art300939) [inline build/js/CardRefresh.js:77]: #3928

File: build/js/CardRefresh.js [hunk 4]
```diff
@@ -65,18 +66,22 @@ class CardRefresh {
...
-      this._removeOverlay()
```
Comment (art300939) [inline build/js/CardRefresh.js:78]: #3927

File: build/js/CardRefresh.js [hunk 5]
```diff
@@ -65,18 +66,22 @@ class CardRefresh {
...
-    }, this._settings.responseType !== '' && this._settings.responseType)
```
Comment (art300939) [inline build/js/CardRefresh.js:79]: #3933

Comment (art300939) [inline build/js/CardRefresh.js:79]: #3934

**Review by art300939 — APPROVED** [review]

**Comment by hkvstore** [comment]
Withdraw

**Pull request closed without merging**